### ISSUE TITLE
Nothing happening

### ISSUE THREAD

**Issue by joh-klein** [AOEpeople/Aoe_Profiler#24]
I followed the rather sort install notes but nothing seems to happen. How do I know, that it is doing anything? Where would I find any output of the profile run?

**Comment by duffybelfield** [comment]
Getting same results, nothing outputting to the frontend :(

**Comment by joh-klein** [comment]
I just saw, that the output is inside a new submenu under "System" (at least it was for me)

**Comment by LeeSaferite** [comment]
Correct. The profiler is storing the profile data to be later viewed via the admin console.

**Comment by colnpanic** [comment]
Look under System > AOE Profiler. I get results there, but for some reason it isn't recording page loads that are more than ~1s long. After a cache flush it has no results until 3 or so refreshes when the load is at ~800ms. This kind of defeats the purpose of profiling slow page loads :grinning: (likely something isn't configured correctly?)

**Comment by adam-paterson** [comment]
@colnpanic have you checked your MySQL error logs? I found that I had to change my available log size because for heavy pages it was trying to write too much data to the database.

**Comment by colnpanic** [comment]
@adam-paterson - hey, yes, after going back and reading the README clear to the end I saw the note about setting "max_allowed_packet=16M" which worked. Well, this particular profiler run was so bad I had to bump it up further, but that fixed it. Thought I had followed up with everywhere I checked into this, thanks for replying here!

**Comment by behrang1981** [comment]
Hiho
i have installed AOE Profiler and see some useful info in backend (admin), but cannot see anything on frontend. Do i miss something?
greets

**Comment by colnpanic** [comment]
@behrang1981 - all the data is in admin now, stored in the database instead of output on the page.

**Comment by behrang1981** [comment]
thanks for reply. can you tell me where in admin? I really don't find it

**Comment by fbrnc** [comment]
Hi @behrang1981, 
please make sure you've installed the latest version from GitHub and all files are in place correctly. After flushing caches and logging in again you should see the Profiler menu item in the System menu.

**Comment by alvinnguyen** [comment]
Hi, is there any reasons why the profiler is now stored in the admin rather than rendering on the front-end.

Thanks for great module :)

**Comment by zeki893** [comment]
can you add this info to the readme.md?

**Comment by aftabnaveed** [comment]
is it possible to still render the profiler on Frontend?  It makes more sense to see the data in real time rather switching back and forth between frontend and backend.

**Comment by predragkondic** [comment]
If someone else can't see any output, check the file permission of var/aoe_profiler.xml :D

**Comment by sharneet** [comment]
I am trying to using this module for profile. I have downloaded Zip file and place onto the magento structure. Enabled Profiler using index.php, but when i navigate to homepage it says "Disable profiler first!". Hitting direct url of product page gives me blank page... Nothing logs into exception files...

It;s just login admin url's

Please help if I am missing something...

**Comment by yoosefi** [comment]
>Copy the aoe_profiler.xml.sample file to var/aoe_profiler.xml

There is no such file: `aoe_profiler.xml.sample`

Documentation is severely lacking in this project.

**Comment by predragkondic** [comment]
> There is no such file: aoe_profiler.xml.sample

The file is created by Varien_Profiler::getConfiguration
The content is placed in: app/code/community/Aoe/Profiler/etc/system.xml

It should be enough to make var/ folder writeable

**Comment by zhernosekim** [comment]
is it possible to still render the profiler on Frontend?

**Comment by Ahmed-Elawadi** [comment]
in this video https://www.youtube.com/watch?v=_usIQ9iW3OI but not working for me, any suggestion?
